README.md

```diff
@@ -52,24 +53,25 @@ Example persistent data paths from [SynthDet](https://github.com/Unity-Technolog
 [Unity Simulation](https://unity.com/products/simulation) provides a powerful platform for running simulations at large scale. You can use the provided cli script to download Perception datasets generated in Unity Simulation:
 
 ```bash
-python -m datasetinsights.scripts.usim_download \
-  --data-root=$HOME/data \
-  --run-execution-id=<run-execution-id> \
+datasetinsights download \
+  --source-uri=<xxx> \
+  --output=$HOME/data \
   --auth-token=<xxx>
 ```
-
-The `auth-token` can be generated using the Unity Simulation [CLI](https://github.com/Unity-Technologies/Unity-Simulation-Docs/blob/master/doc/cli.md#usim-inspect-auth). This script will download the synthetic dataset for the requested [run-execution-id](https://github.com/Unity-Technologies/Unity-Simulation-Docs/blob/master/doc/cli.md#argument-descriptions).
+The `source-uri` is the URI of where this data should be downloaded. Supported source uri patterns ^gs://|^http(s)?://|^usim://. The `output` is the directory on localhost where datasets should be (Default: /data). The `auth-token` can be generated using the Unity Simulation [CLI](https://github.com/Unity-Technologies/Unity-Simulation-Docs/blob/master/doc/cli.md#usim-inspect-auth). This script will download the synthetic dataset for the requested [run-execution-id](https://github.com/Unity-Technologies/Unity-Simulation-Docs/blob/master/doc/cli.md#argument-descriptions).
 
 If the `--include-binary` flag is present, the images will also be downloaded. This might take a long time, depending on the size of the generated dataset.
 
 ### Download SynthDet Dataset
 
-Download SynthDet public dataset from GCS, including GroceriesReal and Synthetic dataset. You can use the provided cli script to download public dataset to reproduce our work.
-
-Here is the command line for GroceriesReal dataset download:
+Download SynthDet public dataset from GCS, including `UnityGroceries-Real` and `UnityGroceries-SyntheticSample` dataset. You can use the provided cli script to download public dataset to reproduce our work. Here is the command line for GroceriesReal dataset download:
```

Comment:
These two dataset are not from GCS downloader instead they use `HttpDownloader`. We should make it clear for users.